README.adoc

```diff
@@ -9,7 +9,7 @@ To install locally:
 
 ----
 
-$ mvn install
+$ mvn install -s .settings.xml
```

Comment:
At the top of the file: `// Do not edit this file (e.g. go instead to src/main/asciidoc)`

This change should be made here: https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/README.adoc